manifests/04_oauthserver_trust_role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
```

Comment:
missing namespace?  The namespace here is known.

Reply:
well spotted